app/models/site_setting.rb

```diff
@@ -0,0 +1,38 @@
+# == Schema Information
+#
+# Table name: site_settings
+#
+#  id         :integer          not null, primary key
+#  title      :string           default("ZeroWaste")
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class SiteSetting < ApplicationRecord
+  acts_as_singleton
+
+  has_one_attached :favicon, dependent: :destroy
+
+  validates :title, presence: true, length: { maximum: 20, minimum: 3 }
+  validates :favicon, attached: true,
+                      content_type: [:png, :jpg, :jpeg, :ico],
+                      size: { less_than: 500.kilobytes,
+                              message: I18n.t("account.site_settings.validations.size") }
+
+  after_initialize :set_default_favicon
+
+  class << self
+    alias_method :current, :instance
```

Comment:
https://stackoverflow.com/questions/17792184/why-alias-method-is-an-instance-method